renderer/src/pages/wallet/GridCanvas.tsx

```diff
@@ -0,0 +1,74 @@
+import { useEffect, useRef } from 'react'
+import { Wallet } from 'src/hooks/StationWallet'
+import { Grid } from 'src/lib/grid'
+import { FILTransactionProcessing } from '../../../../shared/typings'
```

Comment:
can this import path be refactored?

Reply:
Yes, I think it works id I add  a `"shared/*": ["../shared/*"]` entry to tsconfig, then it could be `'shared/typings'`